config/galaxy.ini.sample

```diff
@@ -633,6 +638,29 @@ nglims_config_file = tool-data/nglims.yaml
 # https://FQDN/galaxy-prefix/gie_proxy for proxying
 #dynamic_proxy_prefix=gie_proxy
 
+# The Golang proxy also manages the docker containers more closely than the
+# NodeJS proxy, so is able to expose more container management related options
+
+# This attribute governs the minimum length of time between consecutive HTTP/WS
```

Comment:
This option needs to be IE specific, isn't it? I imagine our PostgreSQL IE that can stay a long time open without any proxy traffic (when a tool loads data into it).